configs/semi/semi_coco_detection.py

```diff
@@ -0,0 +1,124 @@
+# dataset settings
+dataset_type = 'CocoDataset'
+data_root = 'data/coco/'
+file_client_args = dict(backend='disk')
+
+image_size = (1333, 800)
+
+color_space = [
+    [dict(type='AutoContrast')],
+    [dict(type='Equalize')],
+    [dict(type='Invert')],
+    [dict(type='Sharpness')],
+    [dict(type='Posterize')],
+    [dict(type='Solarize')],
+    [dict(type='SolarizeAdd')],
+    [dict(type='Color')],
+    [dict(type='Contrast')],
+    [dict(type='Brightness')],
+]
+
+geometric = [[dict(type='Rotate')], [dict(type='ShearX')],
+             [dict(type='ShearY')], [dict(type='TranslateX')],
+             [dict(type='TranslateY')]]
+
+sup_pipeline = [
```

Comment:
for sup_pipeline, weak_pipeline, strong_pipeline, unsup_pipeline, should add some comments to indicate their specific meanings.